README.md

```diff
@@ -9,6 +9,44 @@
 
 ### How to use
 
+Node (v14):
+
+Run with `node app.js`
+
+app.js:
+```js
+const geostylerSldParser = require("geostyler-sld-parser")
+
+var parser = new geostylerSldParser.SldStyleParser();
+
+const pointSimplePoint = {
+  name: "My Style",
+  rules: [
+    {
+      name: "My Rule",
+      symbolizers: [
+        {
+          kind: "Mark",
+          wellKnownName: "Circle",
+          color: "#FF0000",
+          radius: 6
+        }
+      ]
+    }
+  ]
+};
+
+ 
+parser
+  .writeStyle(pointSimplePoint)
+  .then(function(sld) {
```

Comment:
With `geostyler-style@5` we changed the return value of `readStyle` and `writeStyle` methods (see [geostylerStyle@5 release notes](https://github.com/geostyler/geostyler-style/releases/tag/v5.0.0)).

The new return values use the structure as described in 
- https://github.com/geostyler/geostyler-style/blob/master/style.ts#L755 
- and https://github.com/geostyler/geostyler-style/blob/master/style.ts#L777